contrib/test_jenkins.sh

```diff
@@ -1119,6 +1131,25 @@ run_asan_check() {
 	run_gtest "default"
 }
 
+run_valgrind_check() {
+	if [[ $(uname -m) =~ "aarch" ]] || [[ $(uname -m) =~ "ppc" ]]
+	then
+		echo "==== Skip valgrind tests ===="
```

Comment:
shall we add `uname -a` output to the echo line to provide more details? would it make sense to instead never call `run_valgrind_check` for `aarch` or `ppc`?

Reply:
Added `uname -a` to output.

From my point of view this place for this check is better then [that one](https://github.com/openucx/ucx/pull/9608/files#diff-a938c4972801b864bb71586317dfb5b5eac57b6edf63694b0816154de0ab856bR1165). Or do you propose to move it somewhere else?

Reply:
yes agree. I was thinking moving the check outside of .sh script altogether, maybe to yaml, but that might not be easy